api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -705,11 +713,10 @@ public static final List<GlobalProperty> CORE_GLOBAL_PROPERTIES() {
 		
 		props.add(new GlobalProperty(GP_MAIL_SMTP_STARTTLS_ENABLE, "false",
 		        "Set to true to enable TLS encryption, else set to false"));
-		
-		props.add(new GlobalProperty(GP_HOST_URL, "",
+
+		props.add(new GlobalProperty(GP_PASSWORD_RESET_URL, Context.getAdministrationService().getGlobalProperty(GP_HOST_URL),
```

Comment:
@wikumChamith did you test this by running the web application with this change?

Reply:
@dkayiwa I successfully ran the web application with this change but I couldn't find a way to test this global property directly.

Reply:
What happens to some one who already has a value in `GP_PASSWORD_RESET_URL` and then runs the webapp?

Reply:
I am unable to find a way set a value for it when testing. Could you guide me on that?

Reply:
ok. just remove that and we merge the pull request.

Reply:
should I remove the code that copy `host.url` to `security.passwordResetUrl` ?

Reply:
Correct.

Reply:
I removed it.